python/sparknlp/internal/py4j_utils.py

```diff
@@ -0,0 +1,32 @@
+# Coloring the print
+FAIL = '\033[91m'
+ENDC = '\033[0m'
+
+
+def check_obj_exist(java_class):
```

Comment:
If `java_class` argument gets a datatype different than a string, the function will fail.
We could add a data type validation to make sure `java_class` is a string.